backend/public-graph/graph/resolver.go

```diff
@@ -326,19 +335,35 @@ func (r *Resolver) SubmitSessionEvents(ctx context.Context, sessionID int, event
 		})
 	}
 
-	err := r.DataSyncQueue.Submit(ctx, "", messages...)
+	err := r.DataSyncQueue.Submit(ctx, strconv.Itoa(session.ID), messages...)
```

Comment:
good idea to order the events for a given session, but technically mightt not matter since the inserted events use the timestamp of the event.

Reply:
this was just a small thing I noted while pair programming w/ Spenny - I think it's better to partition by session id in case of an ingestion delay so the session either has all events or no events and isn't missing random ones in between